install/kubernetes/cilium/charts/agent/values.yaml

```diff
@@ -8,3 +8,14 @@ maxUnavailable: 2
 monitor:
   enabled: false
   eventTypes: []
+  # Specifies the resources for the monitor sidecar container if activated
+  resources: {}
+
+# Specifies the resources for the agent container
+resources: {}
+
+# Specifies the resources for the clean-cilium-state init container
+initResources:
+  requests:
+    cpu: "100m"
+    memory: "100Mi"
```

Comment:
Having these requests with default values is hard has it highly depends on the cluster size.

Reply:
cc @tklauser @tgraf

Reply:
Cleaning up local node cilium state depends on cluster size? I would think it's the same set of BPF maps and so on regardless of cluster size. I can understand that the node size could play in, in that we have a number of BPF templates and progs depending on the number of local pods. Are there other aspects you're concerned with?

Reply:
If I understand https://kubernetes.io/docs/tasks/configure-pod-container/assign-memory-resource/ correctly, this is not limiting the memory use but only specifies the requested memory in order to influence scheduling of the pod. IMO `100Mi` memory seems fairly low for a standard Cilium pod.

@maxbischoff how did you determine these numbers?

Reply:
This memory request is only for the init container of the cilium pod, I left the requests for the other containers open to the user on purpose.
We just started using cilium so I cannot tell if having `100Mi` for an initContainer is too little, but with other network plugins (calico) that was plenty in our experience.
I can also remove this default if you would prefer so.

In my opinion, suggesting values for the cilium agent containers is a separate issue from the one I am trying to solve (making the resources configurable)

Reply:
Ah this is for the init container, then I would say this is plenty enough. And like @tklauser said this is the requested memory not the limit so these limits look sane to me

Reply:
Thanks for elaborating. I didn't get that this is only for the init container. In that case LGTM as well.